config/config.go

```diff
@@ -227,6 +228,12 @@ func (c *Config) LoadConfigFromContent(content []byte) (err error) {
 	return
 }
 
+func (c *Config) readCredentialFromEnv() (err error) {
+	c.AccessKeyID = os.Getenv(EnvAccessKeyID)
```

Comment:
What will happen if user only set `access_key_id` without `secret_access_key`?

As proposal said, we need to return error here.

Reply:
It is only set here, and the check function is called later.

Reply:
Got it.